source/style-guide/markup/directives/sharedinclude.txt

```diff
@@ -0,0 +1,234 @@
+.. _style-guide-sharedinclude:
+
+==============
+Shared Include
+==============
+
+The ``sharedinclude`` directive allows you to source content from a file
+located in the MongoDB internal `docs-shared <https://github.com/10gen/docs-shared>`__
+repository. Use this directive when you need to keep the same content in sync
+for multiple sets of documentation or multiple branches of a documentation set.
+
+This page uses the following terms:
+
+- **External file**: content committed to the ``docs-shared`` repository
+  that can be sourced by other documentation repositories.
+- **Sourcing file**: content in a documentation repository that uses a
+  ``sharedinclude`` directive to pull content from an external file.
+- **Placeholder**: a variable that indicates text that the sourcing file
+  must provide a replacement for. These are located in external files and are
+  represented by the placeholder name surrounded by the ``|`` (pipe)
+  character.
+
+.. tip::
+
+   The ``sharedinclude`` directive is an extension of the ``include`` directive.
+   While the ``include`` directive allows you to source content from the
+   branch of the repository in which it is used, the ``sharedinclude``
+   directive is hardcoded to reference external files from the
+   `10gen/docs-shared <https://github.com/10gen/docs-shared/>`__
+   private GitHub repository.
+
+
+Setup
+-----
+
+You must add the URL from which to source the shared content in the
+``sharedinclude_root`` setting of the documentation repository's ``snooty.toml``
+configuration file. This should reference the unprocessed versions of the files,
+currently served from the ``raw.githubusercontent.com`` domain.
```

Comment:
```suggestion
You must add the URL to source the shared content in the
``sharedinclude_root`` setting of the documentation repository's ``snooty.toml``
configuration file from. This URL should reference the unprocessed versions of the files,
currently served from the ``raw.githubusercontent.com`` domain.
```

Reply:
Thanks for rewording this! I think the placement of the preposition at the end of the first sentence may make the sentence more complex, so I'm rewording the paragraph.